source/containers/Form/Form.js

```diff
@@ -16,16 +16,19 @@ const FormContainer = styled.View`
  * is a tool to help you solve the problem of allowing end-users to interact with the
  * data and modify the data in your application.
  */
-function Form({ startAt, steps, firstName, onClose }) {
+function Form({ startAt, steps, firstName, onClose, userAnswer }) {
```

Comment:
Change prop userAnswer to initialAnswers or defaultAnswers. Since it's the answers/values we want to pass to the form, the answers in a general perspective are not dependent on a user, any object could be passed 💃